io-sim/src/Control/Monad/IOSimPOR/Internal.hs

```diff
@@ -1057,7 +1057,9 @@ execAtomically time tid tlbl nextVid0 action0 k0 =
        -> ST s (SimTrace c)
     go ctl !read !written writtenSeq createdSeq !nextVid action = assert localInvariant $
                                                        case action of
-      ReturnStm x -> case ctl of
+      ReturnStm x ->
+        {-# SCC "execAtomically.go.ReturnStm" #-}
```

Comment:
Is this necessary to stay? (The same for the rest of this commit 22a4ed1d56c926bf89038f12a699e213f33823a8)

Reply:
As above.